apps-template/.env

```
@@ -0,0 +1,2 @@
+SENTRY_DSN=https://f51e85c3bd8c51d99059d901a5954113@o4509329568235520.ingest.us.sentry.io/4509329571315712
+ENVIRONMENT=development
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ENVIRONMENT key should go before the SENTRY_DSN key

(UnorderedKey)